Minor guide sections (#6)
ezsystems/commerce-documentation: docs/guide/newsletter/newsletter2go/newsletter2go_cookbook/sending_additional_data_to_newsletter2go.md

# Sending additional data to Newsletter2Go

Before a user is created in the Newsletter2Go address book,
an [event is dispatched](../newsletter2go_service.md) that enables you to send some additional data.

> Any data can be sent to Newsletter2Go, but custom attributes have to be created first in the Newsletter2Go backend.

Reviewer: ```suggestion
To send additional data to Newsletter2Go, you have to also create custom attributes in the Newsletter2Go backend.
```
